operations/accountnotes.md

```diff
@@ -0,0 +1,345 @@
+
+### Response
+
+```javascript
+{
+    "AccountNotes": [
+        {
+            "Id": "a58ff7cb-77e3-495a-bd61-aecf00a3f19d",
+            "AccountId": "1df21f06-0cfc-4960-9c58-a3bf1261663e",
+            "Content": "Brother of the CEO",
+            "AccountType": "Customer",
+            "Classifications": ["FamilyRelations"],
+            "IsActive": true,
+            "CreatorProfile": {
+                "Discriminator": "Enterprise",
+                "EnterpriseProfile": {
+                    "ProfileId": "52d19c34-b0aa-4635-905d-1326fa8b8e13"
+                }
+            },
+            "UpdaterProfile": {
+                "Discriminator": "Integration",
+                "EnterpriseProfile": null
+            }
+        },
+    ],
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AccountNotes` | array of [Account notes](#account-note) | required | Added account notes. |
+
+## Update account notes
+
+> This feature is being actively developed, features and behavior of this operation may change at short notice. Currently we support only one classification per account note. When changing the classification of a note, the current classification must be removed.
+
+Updates information about the specified account notes.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/accountNotes/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "AccountNoteUpdates": [
+        {
+            "AccountNoteId": "a58ff7cb-77e3-495a-bd61-aecf00a3f19d",
+            "Content": {
+                "Value": "The AC in the room doesn't work anymore"
+            },
+            "Classifications": {
+                "Maintenance": {
+                    "Value": true
+                },
+                "Housekeeping": {
+                    "Value": false
+                }
+            },
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `AccountNoteUpdates` | array of [Account note update parameters](#account-note-update-parameters) | required, max 1000 items | Account notes to be updated. |
+
+#### Account note update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AccountNoteId` | string | required | Unique identifier of the account note. |
+| `Content` | [String update value](_objects.md#string-update-value) | optional, max length 1000 characters | Content of the account note \(or `null` if the name should not be updated\). |
```

Comment:
Should that be "\(or `null` if the _content_ should not be updated\)?"

Reply:
It should, it must be a copy-paste mistake. I'll fix it :)